File: sentimental-mario-less/mario.py
# TODO
#Receive height from the user. Height should be between 1 and 8
from cs50 import get_int
while True:
    height = get_int("Enter height between 1 and 8: ")
    if height > 0 and height < 9:
        break


#Code for pyramid
print("Outside of the loop")

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Code to receive height from the user with user validation.
    int n;
    do
    {
        n = get_int("Enter height: ");
    }
    while (n < 1 || n > 8);

    //Code for pyramid

    //for rows
    for (int i = 0; i < n; i++)
    {
        //for columns
        for (int j = 0; j < n; j++)
        {
            if (i + j < n - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}